app/server/lib/GoogleAuth.ts

```diff
@@ -74,7 +74,7 @@ const AUTH_SUBDOMAIN = process.env.GRIST_ID_PREFIX ? `docs-${process.env.GRIST_I
 function getFullAuthEndpointUrl(): string {
   const homeUrl = process.env.APP_HOME_URL;
   // if homeUrl is localhost - (in dev environment) - use the development url
-  if (homeUrl && new URL(homeUrl).hostname === "localhost") {
+  if (homeUrl) {
```

Comment:
What do you think about adding a dedicated variable for this redirect? Something like:
```suggestion
  if (process.env.GRIST_GOOGLE_IMPORT_REDIRECT_URL) {
   return process.env.GRIST_GOOGLE_IMPORT_REDIRECT_URL;
  }

  if (homeUrl && new URL(homeUrl).hostname === "localhost") {
```

and leaving the code as it is? It will be much simpler to manage this integration this way, rather then constructing those URL by hand.

Reply:
@berhalak Do correct me if i'm wrong, but:
in the case of `GRIST_GOOGLE_IMPORT_REDIRECT_URL`, user will need to know what to fill in, whereas,
in the case of just the `APP_HOME_URL`, no intervention is really required.

Reply:
Yes, but:
- user already needs to know what to put in `APP_HOME_URL`, 
- this code that checks localhost is needed, I think, for some tests to work properly,
- this change will probably break some existing installation that relies on this format

With new env variable, this change will be backward compatible, and would require minimal tests change.

Reply:
> With new env variable, this change will be backward compatible, and would require minimal tests change.

No denying that.

But, in that case, if you want the user to also need to take care of the `authHandlerPath` in 
```
  if (homeUrl && new URL(homeUrl).hostname === "localhost") {
    return `${homeUrl}${authHandlerPath}`;
  }
```
then that needs to be documented as well.

My suggestion was to get only the instance URL (which happens to be equal to the `APP_HOME_URL` value) from the user, and append the `authHandlerPath` to that subsequently.

Reply:
Yes, we would need to put the format of that URL in the property description in the `README.md`.

Reply:
The rest of the code in getFullAuthEndpointUrl() does look very specific to the Grist Labs SaaS in its handling of domains. Perhaps, after the local dev case, there could be another case that kicks in for simple (and common) single-domain instances of Grist?